public/less/take-a-stand.less

```diff
@@ -163,75 +192,104 @@
       }
     }
     .pol-pic-link {
-      font-size: 2rem;
+      // font-size: 2rem;
       font-weight: bold;
+      text-align: center;
       .text-font();
       > img {
         border-radius: 50%;
         border: 2px solid @blue;
         height: 18rem;
-        @media (min-width: @medium-bp) and (max-width: @large-bp) {
-          height: 15rem;
+        @media (max-width: @small-bp) {
+          width: 20rem;
+        }
+        @media (min-width: @small-bp) and (max-width: @medium-bp) {
+          height: 17rem;
+        }
+        @media (min-width: @medium-bp) {
+          height: 14rem;
         }
```

Comment:
It ends up smashing the pic: 
https://www.dropbox.com/s/b98v05srpq2y4v5/Screenshot%202016-02-09%2014.03.34.png?dl=0

On Feb 9, 2016, at 1:59 PM, Andy Ernst <notifications@github.com> wrote:

> In public/less/take-a-stand.less:
> 
> >        .text-font();
> >        > img {
> >          border-radius: 50%;
> >          border: 2px solid @blue;
> >          height: 18rem;
> > -        @media (min-width: @medium-bp) and (max-width: @large-bp) {
> > -          height: 15rem;
> > +        @media (max-width: @small-bp) {
> > +          width: 20rem;
> > +        }
> > +        @media (min-width: @small-bp) and (max-width: @medium-bp) {
> > +          height: 17rem;
> > +        }
> > +        @media (min-width: @medium-bp) {
> > +          height: 14rem;
> >          }
> If you want a dynamic width, why not delete height and set width to 100% so that grid framework can scale the size?
> 
> —
> Reply to this email directly or view it on GitHub.
>